analysis/preview.py

```diff
@@ -81,28 +84,41 @@ def parse_positions(jd):
         jd: JSON Document that describes the scenario to be simulated.
 
     Returns:
-        A tuple containing ZSP and Drones positions, respectively.
+        Three arrays containing ZSP positions, Drones positions and Drone trajectories curve step.
     """
     # ZSPs are the easiest
     zsps = jd['ZSPs']
-
     zsp_positions = []
+
     for zsp in zsps:
-        zsp_positions.append(Point(zsp['position']))
+        # check for defined ZSPs position (paper_simple.json need this check)
+        if len(zsp['mobilityModel']['attributes']) > 0 :
+            zsp_positions.append(Point(zsp['mobilityModel']['attributes'][0]['value'])) #Actually ZSPs mobilityModel contains always only one attributes (Position)
```

Comment:
This assumption is too dangerous for future cases. It would be better to search for `position` attribute and then retrieve its value.